.huskyrc.js

```diff
@@ -1,12 +1,12 @@
 /*! m0-start */
 const config = {
     hooks: {
-        'pre-commit': 'lint-staged'
+        'pre-commit':
+            'npm run updateBundle && git add "./test/test-fixtures/transformations/resolved html/p-applicant-british-citizen-or-eu-national.js" && lint-staged'
```

Comment:
In order to keep the test passing the test file needs to be updated with the current version of CW. This pre-commit hook accomplishes that by running a script to update the version and then adding the test file to every commit. This means that if the test file was ever changed but we didn't want to add that change to the commit it would be added anyway

Reply:
I vote that we get rid of the offending test in the first place and not have to do this hack to validate the test. The fact that this needs to be done tells me the test is not great.

Reply:
npm run updateBundle does not need to be run anymore either as there is no filenames or file contents to update